src/Donations/Micros/TaxDeductionCountryModal.tsx

```diff
@@ -74,7 +75,7 @@ export default function TaxDeductionCountryModal(props: any) {
                 value={`${country},${currency}`}
                 onChange={handleCountryChange}
               >
-                {countriesData.map((country: any, index: number) => (
+                {countriesData.map((country: Country[], index: number) => (
```

Comment:
This is incorrect. `country `should be of type `Country`.
Specify type for `countriesData` in the state as below
```
const [countriesData, setCountriesData] = useState<Country[]>([]);
```